gno/r/govdao/messages.gno

```diff
@@ -82,6 +86,6 @@ func NewAddT2MemberMsg(payload *MsgAddT2Member) daokit.ExecutableMessage {
 	return daokit.NewMessage(MsgAddT2MemberKind, payload)
 }
 
-type AddNewT2MemberMessage struct {
+type AddT2MemberMessage struct {
 	Address string
 }
```

Comment:
this should be removed, the correct type is defined above

Reply:
[c17d5db](https://github.com/TERITORI/teritori-dapp/pull/1502/commits/c17d5dbbf4925dd43568129cff8efa0d3ce9e452)